examples/ns-and-soa-management/main.tf

```diff
@@ -0,0 +1,36 @@
+# ---------------------------------------------------------------------------------------------------------------------
+# CONFIGURE OUR AWS CONNECTION
+# ---------------------------------------------------------------------------------------------------------------------
+
+provider "aws" {
+  region = var.aws_region
+}
+
+module "route53" {
+  source = "../.."
+
+  name                         = "test.mineiros.io"
+  skip_delegation_set_creation = true
+
+  records = [
+    {
+      type = "NS"
+      ttl  = 172800
+      records = [
+        "ns-250.awsdns-31.com",
+        "ns-815.awsdns-37.net",
+        "ns-1802.awsdns-33.co.uk",
+        "ns-1290.awsdns-33.org",
+      ]
+    },
+    {
+      type = "SOA"
+      ttl  = 900
+      records = [
+        "ns-250.awsdns-31.com. awsdns-hostmaster.amazon.com. 1 7200 900 1209600 86400"
+      ]
+    }
+  ]
```

Comment:
those are not editable.. as defined by the DNS afaik.. so what is the purpose?

Reply:
resolved in 30ae3f7